gnomad/utils/annotations.py

```diff
@@ -1099,7 +1159,7 @@ def _sum_or_diff_fields(
     callstat_ann_af = ["AC", "AF", "AN", "homozygote_count"]
     new_freq = freq_meta_idx.map(
         lambda x: hl.bind(
-            lambda y: y.annotate(AF=hl.or_missing(y.AN > 0, y.AC / y.AN)).select(
+            lambda y: y.annotate(AF=hl.if_else(y.AN > 0, y.AC / y.AN, 0)).select(
```

Comment:
what's this change for?

Reply:
I don't recall touching this manually, but I looked back and this change occurred when I was copying in the code from the old branch and setting up the PR. Apparently something was either ahead or behind of something else.

Reply:
but anyways I reverted it back to the or_missing() functionality